### ISSUE TITLE
[Bug Report] No Neuron window appears on v0.114.0 and higher.

### ISSUE THREAD

**Issue by jordanmack** [nervosnetwork/neuron#3135]
### Topic Type

Other

### Detail of the issue

Installing v0.114.0, v0.114.2, and v0.114.3 result in no viewable window when it is run. The process starts but nothing happens. Force quitting and rebooting does not help. Reverting to v0.112.0 fixes the problem. 

![20240419155256](https://github.com/nervosnetwork/neuron/assets/37931/176ab8bc-6cae-4eb9-9471-fc0089b4f254)

Windows 11 23H2 Build 22631.3447
Intel x64 + 32GB RAM
Neuron installed locally (for just me).
Configured to use a CKB full node on the local network but a different computer.

### bundled-ckb.log

```shell
N/A
```

### bundled-ckb-light-mainnet.log

```shell
N/A
```

### main.log

```shell
[2024-04-19T23:22:59.009Z] [info]  App:	Neuron is starting
```

### status.log

```shell
N/A
```

**Comment by Keith-CY** [comment]
Hi @jordanmack 

It would be helpful if we could debug with the log file of Neuron.

The log is named `main.log` and can be found in the directory `%AppData%/Neuron/logs`, the directory can be visited by the `File Explorer`.

**Comment by jordanmack** [comment]
I already gave the complete log. This is all that exists when this problem occurs.

**Comment by jordanmack** [comment]
This seems to be fixed in 0.116.2.
